sync/src/chain.rs

```diff
@@ -1300,6 +1301,13 @@ impl ChainSync {
 		// TODO [todr] propagate transactions?
 	}
 
+	/// Add transaction to the transaction queue
+	pub fn insert_transaction<T>(&self, transaction: ethcore::transaction::SignedTransaction, fetch_nonce: &T)
+		where T: Fn(&Address) -> U256
+	{
+		let mut queue = self.transaction_queue.lock().unwrap();
+		queue.add(transaction, fetch_nonce);
```

Comment:
add returns a result right now so this will be a warning (after merge with master). We should forward the result up to `sync/src/lib.rs`